clis/dtcli/src/dtcli.rs

```diff
@@ -0,0 +1,113 @@
+/// Code for the daemon which runs actual libtelio in the background.
+use anyhow::Result;
+
+use tokio::task::JoinHandle;
+
+use telio_task::io::{wait_for_tx, Chan};
+
+use crate::coms::{DaemonIPC, IPCMessage};
+use crate::get_ipc_socket_path;
+use crate::telio_cli_interface::TelioCliInterface;
+
+/// Function to setup the DTCLI daemon's dependencies (including the async runtime) and and run it.
+///
+/// # Arguments
+///
+/// * `features` - Optional string containing the features specification for Telio in JSON.
+pub fn run_dtcli(features: Option<String>) -> Result<()> {
+    let nord_token = std::env::var("NORD_TOKEN").ok();
```

Comment:
Does this function? can we assume deamon that is ran with root, coppies env vars with fork?

Reply:
It works and a forked process inherits the exported environment variables. I did update the runner to use sudo -E tho instead of just sudo. I used to pass the token through function arguments at an earlier stage of development, I can bring that back if you think it'll make the code more robust against issues with the environment.